nncf/common/tensor_statistics/statistic_point.py

```diff
@@ -71,6 +71,22 @@ def add_statistic_point(self, statistic_point: StatisticPoint) -> None:
 
             self.data[target_node_name].append(statistic_point)
 
+    def remove_statistic_point(self, algorithm: str) -> None:
+        """
+        Method to remove statistic point associated with a given algorithm
+        from statistic point container.
+
+        :param algorithm: Algorithm name
+        """
+        for target_node_name, statistic_points in list(self.data.items()):
```

Comment:
Why do you need conversion to list?

Reply:
Pardon! I don't need it, just forgot to remove it ;-; 
Will remove it shortly